keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -508,6 +755,10 @@ devices:
         int: 0
       uuid:
         string: GPU-4cf8db2d-06c0-7d70-1a51-e59b25b2c16c
+    consumesCapacity:
```

Comment:
Not really about this line...

Each of the GPUs is shown with attributes "index" and "minor". I think having just one per-device attribute (the "uuid") is sufficient and easier to understand.

As it stands now, as I reader I am trying to understand what "index" and "minor" mean although I shouldn't bother yet.

Reply:
yes, for the example, no need to overcomplicate it I guess...